data/json/harvest.json

```diff
@@ -17,6 +17,7 @@
   {
     "id": "mon_broken_cyborg",
     "type": "harvest",
+    "message": "You search for any salvagable bionic hardware in this mess of flesh and steel",
```

Comment:
I'm not sure about ```flesh``` in the message. Cyborg's description says it's ```a robot body with the head of a human``` and also mentioning only some patches of rotting skin.